pageserver/src/http/routes.rs

```diff
@@ -183,8 +183,8 @@ impl From<crate::tenant::DeleteTimelineError> for ApiError {
         use crate::tenant::DeleteTimelineError::*;
         match value {
             NotFound => ApiError::NotFound(anyhow::anyhow!("timeline not found")),
-            HasChildren => ApiError::BadRequest(anyhow::anyhow!(
-                "Cannot delete timeline which has child timelines"
+            HasChildren(children) => ApiError::BadRequest(anyhow::anyhow!(
```

Comment:
isn't this more of PreconditionFailed or Conflict ?
the request contents is valid but the server state of the timeline forbids it

Reply:
Makes sense, changed